packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -106,7 +113,7 @@ export function TimeRangePicker(props: TimeRangePickerProps) {
       <Tooltip content={<TimePickerTooltip timeRange={value} timeZone={timeZone} />} placement="bottom" interactive>
         <ToolbarButton
           data-testid={selectors.components.TimePicker.openButton}
-          aria-label={`Time range picker with current time range ${formattedRange(value, timeZone)} selected`}
+          aria-label={`${currentTimeRangeMessage} ${formattedRange(value, timeZone)} ${selectedMessage}`}
```

Comment:
@JoaoSilvaGrafana @joshhunt, I'm not sure about this. Is it ok? 🤔

Reply:
We should merge these messages so it is easier for the translator to have the full context. Like in the last example of this section: https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#plain-js-usage